packages/rn-tester/e2e/__tests__/Alert-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+/* global device, element, by, expect, waitFor */
```

Comment:
This shouldn't be necessary because they're configured in the eslint/flow settings. Is that not working?

Reply:
No, it is not working.

<img width="693" alt="Screenshot 2020-09-10 at 9 58 07 PM" src="https://user-images.githubusercontent.com/22813027/92762961-b7b35b80-f3b0-11ea-8014-6cb6729cfc5a.png">

Reply:
`element` may not be configured but `device` and `expect` should be?